packages/core/test/new-api/staticCall.ts

```diff
@@ -0,0 +1,156 @@
+/* eslint-disable import/no-unused-modules */
+import { assert } from "chai";
+
+import { buildModule } from "../../src/new-api/build-module";
+import { NamedStaticCallFutureImplementation } from "../../src/new-api/internal/module";
+import { FutureType } from "../../src/new-api/types/module";
+
+describe("static call", () => {
```

Comment:
Can we add a test showing the result of a static call being used as the input to another call?